linked_list/.~c9_invoke_3jhEI2.py

```diff
@@ -0,0 +1,110 @@
+class AbstractLinkedList(object):
+    """
+    Abstract class representing the LinkedList inteface you must respect.
+    
+    You must not implement any of the method in this class, and this class
+    must never be instantiated. It's just a "guide" of which methods
+    the LinkedList class should respect.
+    """
+
+    def __str__(self):
+        raise NotImplementedError()
+
+    def __len__(self):
+        raise NotImplementedError()
+
+    def __iter__(self):
+        raise NotImplementedError()
+
+    def __getitem__(self, index):
+        raise NotImplementedError()
+
+    def __add__(self, other):
+        raise NotImplementedError()
+
+    def __iadd__(self, other):
+        raise NotImplementedError()
+
+    def __eq__(self, other):
+        raise NotImplementedError()
+
+    def append(self, element):
+        raise NotImplementedError()
+
+    def count(self):
+        raise NotImplementedError()
+
+    def pop(self, index=None):
+        raise NotImplementedError()
+
+
+class Node(object):
+    """
+    Node class representing each of the linked nodes in the list.
+    """
+
+    def __init__(self, elem, next=None):
+        self.elem = elem
+        self.next = next
+
+    def __str__(self):
+        return str(self.elem)
+
+    def __eq__(self, other):
```

Comment:
I personally wasn't able to do much, but from what I understood I wonder if you have to make not just the node equal but its connection also equal.